app.js

```diff
@@ -68,7 +75,17 @@ const csrf_options = {
 };
 const csrf_middleware = csrf(csrf_options); //initialize and return middleware
```

Comment:
If you want I believe you can use app.use(csrf_middleware) to apply across all routes….but not essential!